### PR TITLE
DIV-4520: Created helm charts & liveliness check

### PR DESCRIPTION
**Pull request by johnoneill46**

### CODE REVIEW

File: charts/div-emca/values.aat.template.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    DOCUMENT_MANAGEMENT_STORE_URL: "http://dm-store-aat.service.core-compute-aat.internal"
+    IDAM_API_HEALTH_URI: "https://idam-api.aat.platform.hmcts.net/health"
+    HTTP_CONNECT_TIMEOUT: "60000"
+    APPLICATION_INSIGHTS_IKEY: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
+    HTTP_CONNECT_SOCKET_TIMEOUT: "1000"
+    AUTH_PROVIDER_SERVICE_CLIENT_BASEURL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+    SLOT: "PRODUCTION"
+    REFORM_SERVICE_NAME: "emca"
+    IDAM_API_URL: "https://idam-api.aat.platform.hmcts.net"
+    REFORM_TEAM: "div"
```
Comment (qzhou-hmcts) [inline charts/div-emca/values.aat.template.yaml]: can you move  the following variables to  values.template.yaml as they are not environment based.

- REFORM_TEAM
- REFORM_SERVICE_NAME
- HTTP_CONNECT_SOCKET_TIMEOUT
- HTTP_CONNECT_TIMEOUT
- HTTP_CONNECT_REQUEST_TIMEOUT
- AUTH_PROVIDER_SERVICE_CLIENT_TOKENTIMETOLIVEINSECONDS
- AUTH_PROVIDER_SERVICE_CLIENT_MICROSERVICE
- AUTH_PROVIDER_SERVICE_CLIENT_KEY

File: charts/div-emca/values.aat.template.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+java:
+  environment:
+    APPINSIGHTS_INSTRUMENTATIONKEY: "63c30ef9-a27b-4d38-8afc-4c180fb6ac33"
+    AUTH_PROVIDER_SERVICE_CLIENT_KEY: ${AUTH_PROVIDER_SERVICE_CLIENT_KEY}
```
Comment (qzhou-hmcts) [inline charts/div-emca/values.aat.template.yaml]: Can you use load the secret into the volume in the same way as this PR 
https://github.com/hmcts/div-document-generator-client/pull/113/files#diff-c5b8eff8c293e4bfe71889dc3c323f38R13

File: charts/div-emca/values.template.yaml [hunk 3]
```diff
@@ -4,6 +4,17 @@ java:
...
+    AUTH_PROVIDER_SERVICE_CLIENT_TOKENTIMETOLIVEINSECONDS: "900"
+    HTTP_CONNECT_SOCKET_TIMEOUT: "1000"
+    HTTP_CONNECT_TIMEOUT: "60000"
+    HTTP_CONNECT_REQUEST_TIMEOUT: "60000"
+    SLOT: "PRODUCTION"
+keyVaults:
+  "div":
+    resourceGroup: div
+    secrets:
+      - div-emca-s2s-auth-secret
```
Comment (tchow8) [inline charts/div-emca/values.template.yaml]: Don't think this exists.
Check div vaults, but I think we have been using 'div-doc-s2s-auth-secret' for EMCA service auth too.

File: src/integrationTest/resources/application.properties [hunk 4]
```diff
@@ -28,3 +28,4 @@ idam.auth.microservice=divorce_document_generator
 ###############################################
 evidence.management.client.api.baseUrl=#{environment["TEST_URL"].replaceFirst("https://", "http://")}/emclientapi/version/1
 evidence.management.client.api.endpoint.upload=/upload
+spring.main.allow-bean-definition-overriding=true
```
Comment (hosainnet) [inline src/integrationTest/resources/application.properties]: Why did we need this?

**Comment by hosainnet** [comment]
No longer needed as #97 was merged

**Pull request closed without merging**